akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/RollingUpdateShardAllocationSpec.scala

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import akka.actor.Actor
+import akka.actor.ActorLogging
+import akka.actor.Address
+import akka.actor.Props
+import akka.cluster.Cluster
+import akka.cluster.MemberStatus.Up
+import akka.serialization.jackson.CborSerializable
+import akka.testkit.ImplicitSender
+import com.typesafe.config.ConfigFactory
+import scala.concurrent.duration._
+
+object RollingUpdateShardAllocationSpecConfig
+    extends MultiNodeClusterShardingConfig(
+      additionalConfig = """
+      akka.cluster.sharding {
+        # speed up forming and handovers a bit
+        retry-interval = 500ms
+        waiting-for-state-timeout = 500ms
+        rebalance-interval = 1s
+        # we are leaving cluster nodes but they need to stay in test
+        akka.coordinated-shutdown.terminate-actor-system = off
```

Comment:
have you tried this test with the new allocation strategy?
```
akka.cluster.sharding.least-shard-allocation-strategy.rebalance-absolute-limit = 1
```

Reply:
I have not, we should have it for both, (or perhaps only for the new, now that we intend to deprecate the old)

Reply:
Switched over to cover the new one (after running the old one with the latest updates and fixing some instability)